js/components/GoogleColumnChart.jsx

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+var $ = require('jquery');
+var PubSub = require('pubsub-js');
+var React = require('react');
+
+var GoogleColumnChart = React.createClass({
+
+  getInitialState: function() {
+    return { data: [] };
+  },
+
+  componentDidMount: function() {
+    this.fetchData(this.props);
+    this.drawCharts();
+    PubSub.subscribe('googlecharts', this.drawCharts);
+  },
+
+  componentWillReceiveProps: function(newProps) {
+    this.fetchData(newProps);
+  },
+
+  componentDidUpdate: function() {
+    this.drawCharts();
+  },
+
+  fetchData: function(props) {
+    var user = props.user;
+    var category = props.category;
+
+    var url = '/api/usersnapshots'
+      + '/' + user._id
+      + '/' + category
+      + '/' + this.props.usertype
+      + '/' + this.props.datatype;
+    $.ajax({
+      url: url,
+      success: function(data) {
+        this.setState({ data: data });
+      }.bind(this),
+      error: function(xhr, status, err) {
+        console.error(xhr.responseText);
+      }.bind(this)
+    });
+  },
+
+  drawCharts: function() {
+    // Do nothing if google is not initialized
+    if (!googleLoaded || this.state.data.length === 0) {
```

Comment:
I think it doesn't make as much sense to be a state variable, since it's a global that starts when google loads in index.html. Leaving it as a global allows it to be used in the future by anything that might want to use the charts